spec/asyncapi.md

```diff
@@ -645,7 +645,9 @@ subscribe:
 
 #### <a name="operationObject"></a>Operation Object
 
-Describes a publish or a subscribe operation. This provides a place to document how and why messages are sent and received. For example, an operation might describe a chat application use case where a user sends a text message to a group.
+Describes a publish or a subscribe operation. This provides a place to document how and why messages are sent and received.
+
+For example, an operation might describe a chat application use case where a user sends a text message to a group. A publish operation describes messages that are received by the chat application, whereas a subscribe operation describes messages that are sent by the chat application.
```

Comment:
Hello. New here but willing to collaborate.
While reading this, I think "chat application" is quite ambiguous of who's sending and receiving. Following to the [Event Driven Architectures docs](https://www.asyncapi.com/docs/getting-started/event-driven-architectures), it might be clearer to state "publish operation describes messages that are received by the chat application's message broker" and "subscribe operation describes messages that are sent by the chat application's message broker", or around those lines

Reply:
hi @ alexleonescalera, i agree that chat is not the best example, but this is the example currently used in the spec in this location. I wanted to provide a focused point change to the spec that just provides more clarity on the intent of the `publish` and `subscribe` operations - without making any other changes, such as changing the example used here.